JNIBridge.cpp

```diff
@@ -103,12 +104,34 @@ static void ClearErrors()
 	}
 }
 
+jclass DefaultFindClass(JNIEnv* env, const char* name);
+
+void ResetOverrides()
+{
+	g_Overrides.FindClass = DefaultFindClass;
+}
+
 // --------------------------------------------------------------------------------------
 // Initialization and error functions (public)
 // --------------------------------------------------------------------------------------
-void Initialize(JavaVM& vm)
+void Initialize(JavaVM& vm, CallbackOverrides* overrides)
 {
 	g_JavaVM = &vm;
+
+	if (overrides != NULL)
+	{
+		g_Overrides = *overrides;
```

Comment:
Missing fixup here. You should check if overrides actually have a function set and fallback to default if it does not.

Reply:
https://github.com/Unity-Technologies/android-jni-bridge/commit/5bd1c756d274bb5e324b9c23d165edd9f5814a87